HideSection.hooks.php

```diff
@@ -59,32 +59,36 @@ public static function onSkinEditSectionLinks( $skin, $title, $section, $tooltip
 
     }
 
-    public static function onSkinTemplateOutputPageBeforeExec ( &$skin, &$template ) {
+    public static function onBeforePageDisplay( OutputPage $out, Skin $skin ) {
         global $wgHideSectionTitleLink;
 
-        if ($wgHideSectionTitleLink) {
+        if ( $wgHideSectionTitleLink ) {
             $showall  = wfMessage( 'hidesection-showall' )->text();
             $hideall  = wfMessage( 'hidesection-hideall' )->text();
             $titleall = wfMessage( 'hidesection-hidealltitle' )->text();
 
-            $linkelem = Html::element('a', [
-                    "class" => "hidesection-all internal",
-                    "data-show" => $showall,
-                    "data-hide" => $hideall,
-                    "title" => $titleall,
-                    "href" => "#`"
+            $linkelem = Html::element( 'a', [
+                    'class' => 'hidesection-all internal',
+                    'data-show' => $showall,
+                    'data-hide' => $hideall,
+                    'title' => $titleall,
+                    'href' => "#`"
                 ],
                 $hideall
             );
-            $hideelem = Html::Rawelement('span',
-                 array( 'class' => 'hidesection-head' ),
+            $hideelem = Html::Rawelement( 'span',
+                 [ 'class' => 'hidesection-head' ],
                 '[' . $linkelem . ']'
             );
 
+            $title = $out->getPageTitle();
+    		if ( !$title instanceof Title ) {
+    			return;
+    		}
+
             // Append to page title
-            $template->data['title'] .= $hideelem;
+            $out->setPageTitle( $title . ' ' . $hideelem );
```

Comment:
Did you test this change?  You added whitespace here, which does have layout effects in HTML.

Reply:
No, I tried to find an example on how to migrate and found something that looked like it was related.